R/family.R

```diff
@@ -43,29 +43,66 @@ epidist_family_model.default <- function(data, family, ...) {
 #' Reparameterise an `epidist` family to align `brms` and Stan
 #'
 #' @inheritParams epidist_family
-#' @rdname epidist_family_reparam
+#' @rdname epidist_family_param
 #' @family family
 #' @export
-epidist_family_reparam <- function(family, ...) {
-  UseMethod("epidist_family_reparam")
+epidist_family_param <- function(family, ...) {
```

Comment:
I left this as it is in case the default parame strategy doesn't work in all cases (i.e you can write a custom workaround). I thought we would monitor this and take it out if it appears to work well.